components/wifi_connector/wifi_station.cpp

```diff
@@ -0,0 +1,88 @@
+#include "wifi_station.hpp"
+
```

Comment:
Nitpick: This newline can be removed, and the includes ordered lexicographically.